lang/cs/Org.Apache.REEF.IO.Tests/TestHadoopFileSystem.cs

```diff
@@ -163,10 +164,38 @@ public void CreateUriForPathWithPrefix()
             Assert.Equal(new Uri(uriString), _fileSystem.CreateUriForPath(uriString));
         }
 
+        /// <summary>
+        /// Tests whether .IsDirectory() works.
+        /// </summary>
+        [Fact(Skip = SkipMessage)]
+        public void TestIsDirectory()
```

Comment:
I would split that test into three independent tests - one for each condition

Reply:
fixed

---
In reply to: [198682763](https://github.com/apache/reef/pull/1473#discussion_r198682763) [](ancestors = 198682763)